backend/workflow_manager/endpoint/destination.py

```diff
@@ -409,3 +415,42 @@ def get_json_schema_for_api(cls) -> dict[str, Any]:
             os.path.dirname(__file__), "static", "dest", "api.json"
         )
         return cls.get_json_schema(file_path=schema_path)
+
+    def _push_to_queue(
+        self,
+        file_name: str,
+        error: Optional[str] = None,
+        result: Optional[str] = None,
+    ) -> None:
+        """Handle the QUEUE result.
+
+        This method is responsible for pushing the input file and result to
+        review queue.
+        Args:
+            file_name (str): The name of the file.
+            result (Optional[str], optional): The result of the API call.
+                Defaults to None.
+
+        Returns:
+            None
+        """
+        connector: ConnectorInstance = self.endpoint.connector_instance
+        connector_settings: dict[str, Any] = connector.connector_metadata
+        queue_result = {"file": file_name}
+        if error:
+            return
```

Comment:
@vishnuszipstack do we log in cases of error? Also please move this to the start of the function to exit early - we can avoid making DB queries in case of an error

1. Why do we pass the error argument if its not going to be used?
1. Do we wish to push and show the error causing files in manual review queue screen?

Reply:
@chandrasekharan-zipstack error not required. While copying existing code for modification, missed to remove this. Now removed.